source/agora/consensus/EnrollmentManager.d

```diff
@@ -22,17 +22,17 @@
     In order to ensure that we will never loose the ability to reveal
     pre-images, this implementation uses a reproducible scheme:
     on the first run, we generate a "cycle seed" which is derived from a hash
-    of the private key, a constant string, and a nounce, starting from 0.
+    of the private key, a constant string, and a nonce, starting from 0.
```

Comment:
In the UK a `nonce` is also an insult. For example "You're such a nonce" means "you're so stupid".

_The More You Know...!_